typing/solver.ml

```diff
@@ -576,7 +1017,9 @@ module Solver_mono (C : Lattices_mono) = struct
   let zap_to_ceil : type a l. a C.obj -> (a, l * allowed) mode -> log:_ -> a =
    fun obj m ~log ->
     let ceil = get_ceil obj m in
-    submode obj (Amode ceil) m ~log |> Result.get_ok;
+    (* We want a hint to explain why [ceil] is high. However, we only have hint
```

Comment:
Could we introduce a special hint for zapping here? Same below.

Reply:
Yes we should, there is a CR in solver_intf about  this.